libraries/hotstuff/include/eosio/hotstuff/state.hpp

```diff
@@ -0,0 +1,61 @@
+#include <eosio/chain/block_header.hpp>
+
+#include <eosio/hotstuff/qc_chain.hpp>
+
+namespace eosio::hotstuff {
+
+    using namespace eosio::chain;
+
+    struct safety_state {
+
```

Comment:
Persistent files fork_db and snapshot in nodeos each has a magic number to help make sure the file is not corrupted. Should consider using one for safety state file.

Reply:
Prepended a magic uint64_t to state files.

Reply:
I don't see where you added the magic number. It can be added later.